pkg/sbproxy/reconcile/resyncer.go

```diff
@@ -93,21 +87,43 @@ func (r *resyncJob) process(ctx context.Context) {
 	}
 	logger.Infof("resyncJob successfully retrieved %d brokers from platform", len(platformBrokers))
 
-	r.reconcileBrokers(ctx, platformBrokers, smBrokers)
+	r.reconcileBrokers(ctx, platformBrokers, allBrokers)
 	r.resetPlatformCache(ctx)
-	r.reconcileVisibilities(ctx, smVisibilities, smBrokers)
+	r.reconcileAllVisibilities(ctx, allBrokers, allBrokersPlans)
 }
 
-func (r *resyncJob) getSMPlans(ctx context.Context, smBrokers []*platform.ServiceBroker) (map[string]brokerPlan, error) {
+func (r *resyncJob) reconcileAllVisibilities(ctx context.Context, allBrokers []*platform.ServiceBroker, allBrokersPlans map[string]map[string]brokerPlan) {
+	logger := log.C(ctx)
+	if r.options.VisibilityBrokerChunkSize > 0 {
+		brokerChunks := chunkSlice(allBrokers, r.options.VisibilityBrokerChunkSize)
```

Comment:
maybe better 
chunkSize := r.options.VisibilityBrokerChunkSize
if chunkSize == 0 -> chunkSize = len(allBrokers)
and then it'll be single implementation

Reply:
done